docs/source/porting_guide/PORTING_2.rst

```diff
@@ -0,0 +1,62 @@
+------------------------
+Receptor 2 Porting Guide
+------------------------
```

Comment:
I think you only need the heading format underneath the text. This also applies to the section headings below.

Reply:
It's a preference (see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections which lists it as overlining)